src/components/TodoList/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import TodosCounter from '../TodosCounter';
+import TodoCreator from '../TodoCreator';
+
+function TodoList() {
+    const [newTodo, setNewTodo] = useState('')
+    const [todos, setTodos] = useState([])
+    useEffect(() => {
+        const initialTodos = [
+            'hacer ejercicio',
+            'tender la cama',
+            'desayunar',
+            'estudiar',
+            'hacer postwork'
+        ]
+        setTodos(initialTodos)
+    }, [])
+
+    function onUpdateTodo(value) {
```

Comment:
Use arrow function syntax instead of function syntax when you are adding function handlers inside the component